src/_includes/interactive-controls.html

```diff
@@ -14,7 +14,7 @@ <h2>Axes & Instances</h2>
 				class="interactive-controls-slider"
 			/>
 		</li>
-		{% for axis in axes %}
+		{% for axis in font[1].axes %}
```

Comment:
Should this loop over all fonts? And why `[1]` and not `[0]`?

Reply:
It's the way 11ty returns data in `_data`: `[0]` contains the directory name, and `[1]` contains the actual data for each file. (In this case, `axes.json`). The looping happens [here](https://github.com/kabisa/specimen-skeleton/pull/58/files#diff-e249faefed5757034596c5096d33dab6R15) (prettier smashes this into one line, not happy about that, but hopefully you can parse it ;-))

Reply:
Ah clear!